config/config.yaml

```diff
@@ -3,6 +3,8 @@ default:
         history:
             filename: ../../cmdhist.pcap
         port: 3075
+        topic: __commands__
+        zmq_conn_wait: 1
```

Comment:
I think we can drop these from the default config and stick with documenting them only. We pull defaults for them and I really can't imagine a situation where 99% of users want to change them. The slimmer the default config the better.